config.php

```diff
@@ -139,6 +139,11 @@
 	// won't be displayed if there is only one item (rookgaard for example)
 	'character_towns' => array(1),
 
+	// characters lenght
+	// This is the mininum and the maxinum length that a player can create a character. It is highly recommend the maxinum lenght be 21.
+	'min_length' => 4,
+	'maxinum_length' => 21
```

Comment:
I would change it to "character_name_max_length", so it's more clear what it does.